substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -652,20 +676,21 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedDepositWeight: Get<BalanceOf<Self>>;
 
-		/// The maximum number of winners that can be elected by this `ElectionProvider`
-		/// implementation.
+		/// Maximum number of winners that an election supports.
 		///
 		/// Note: This must always be greater or equal to `T::DataProvider::desired_targets()`.
-		#[pallet::constant]
 		type MaxWinners: Get<u32>;
 
```

Comment:
Won't the staking miner binary need `MaxWinners` and `MaxBackersPerWinner`? maybe they should be in the metadata then.